www/dashboard/payday.spt

```diff
@@ -0,0 +1,77 @@
+import calendar
+import datetime
+from decimal import Decimal as D
+
+[---]
+if user.ADMIN:
```

Comment:
I would say invert this since it's not a part of a page (as before) but the entire page.

```
if not user.ADMIN:
    raise Response(401 if user.ANON else 403)
```